build/common.go

```diff
@@ -99,7 +99,12 @@ func buildBackend(cfg Config) error {
 	}
 
 	// TODO: Change to sh.RunWithV once available.
-	return sh.RunWith(cfg.Env, "go", args...)
+	err = sh.RunWith(cfg.Env, "go", args...)
+	if err != nil {
+		return err
+	}
+	b := Build{}
+	return b.GenerateManifestFile()
```

Comment:
Since `GenerateManifestFile` is now part of `buildBackend` (which is called in `Linux`, `Windows`, etc), can we also remove it from `BuildAll`?

https://github.com/grafana/grafana-plugin-sdk-go/blob/549e0f04b019c19373c4522d6e0e02ea865caa84/build/common.go#L343